MANIFESTO/README.md

```diff
@@ -0,0 +1,47 @@
+# The Open Source Consumption Manifesto
+_v 0.0 (public draft)_
+
+Open source is simultaneously a public good and a good of the public.
+
+Open source doesn’t owe anything to an individual or organization.  
+
+The availability, value, and quality of open source are not guaranteed.
+
+### As consumers of open source, we are responsible for *the open source we use*, *how we consume it*, and *how we manage the risk associated with that consumption*.
+
+### We seek to...
+
+1. Prioritize secure consumption of open source components
+2. Be aware and considerate of the developer experience
+3. Build upon iterative policy-based foundations and best practices. 
+
+### We call on commercial and non-commercial development organizations to…
+
+* Accept open source software consumption as critical to building a secure software supply chain.
+<br>
+
+* Ensure that open source software consumption is balanced against a defined risk profile which can depend on risk tolerance, regulatory context, etc.
+<br>
+
+* Recognize potential risks associated with open source consumption, including vulnerabilities and malicious software.
```

Comment:
```suggestion
* Recognize potential risks associated with open source consumption, including inappropriate choices, vulnerabilities and malicious software.
```
I'm not certain about my word choice in this suggestion, but I do think it's worth calling out that some of the risks for open source consumption are not malicious, but are rather because the chosen component is not an appropriate choice (for whatever reason).